docs/content/docs/start/tutorial.md

```diff
@@ -7,188 +7,154 @@ aliases:
+In this tutorial we will start with a dice roller that works without Fluid locally, and then show how to add collaboration by changing a few key pieces. 
 
-## Connect container to service for collaboration
+This `renderDiceRoller`, given an HTML element to attach to, creates a working dice roller that displays a random dice value each time the "Roll" button is clicked. Note that demo code ommits the styles for brevity.
 
-To orchestrate the collaboration, you need to connect to a service to send and receive the updates to the data. To do this, connect a [Fluid container][] object to the service and load your container code into it.
+```js
+function renderDiceRoller(elem, diceMap) {
+    const dice = document.createElement("div");
 
-For now, run on a local test service called Tinylicious, and to make it easier to connect to this service we've provided a helper function `getTinyliciousContainer()`. The helper function takes a unique ID to identify your **document** (the collection of data used by your app), the container code, and a flag to indicate whether you want to create a new document or load an existing one. Use any app logic you'd like to generate the ID and determine whether to create a new document. In the [example repository](https://github.com/microsoft/FluidHelloWorld/blob/main/src/app.ts), we use the timestamp and URL hash as just one way of doing it.
+    const rollButton = document.createElement("button");
+    rollButton.textContent = "Roll";
+    rollButton.onclick = () => updateDice(Math.floor(Math.random() * 6));
+    
+    elem.append(dice, rollButton);
 
-```ts
-const container =
-    await getTinyliciousContainer(documentId, DiceRollerContainerRuntimeFactory, createNew);
+    const updateDice = (value) => {
+        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
+        dice.textContent = String.fromCodePoint(0x2680 + value);
+    };
+    updateDice(1);
+}
 ```
 
-This will look a little different when moving to a production service, but you'll still ultimately get a reference to a `Container` object running your code and connect to a service.
+### Modifying Fluid data
 
-After you've connected the `Container` object, your container code will have already run to create an instance of your model. Because you used a `ContainerRuntimeFactoryWithDefaultDataStore` to build your container code, you can also use a helper function Fluid provides called `getDefaultObjectFromContainer` to get a reference to the model instance.
+The first thing to change is what happens when the user clicks the `rollButton`. Instead of having the button update the state directly, the button updates the `value` key on the passed in `diceMap`. This creates a new op on the client which will be communicated with all of the other collaborators. Pushing local state out to Fluid state is a common pattern because the view should react to local and remote changes in the same way.
```

Comment:
```suggestion
To begin using Fluid in the application, the first thing to change is what happens when the user clicks the `rollButton`. Instead updating the local state directly, the button updates the `value` key on the passed in `diceMap`. This change will be distributed to all clients and cause a `valueChanged` event to be sent. The event handler will then update the view. This pattern is common in Fluid because the view should react to local and remote changes in the same way.
```